### ISSUE TITLE
Teensy 3.5 led strip matrix

### ISSUE THREAD

**Issue by MAXimumEFFORT843333** [TylerTimoJ/LMCSHD#2]
Hello,

using a 25x25 matrix ws2812B led strips, snake top left and horizontal. the program isnt changing anything on the teensy. ive connected to the program before this patch so not sure why its not connecting now

**Comment by TylerTimoJ** [comment]
Do you mean that nothing is being displayed, or is it something to do with the pixel order? Make sure you don't have anything else using the serial port, like the Arduino serial monitor.

**Comment by MAXimumEFFORT843333** [comment]
Nothing is being displayed. Ive set the pixel order correctly. 

It could be the serial port? In an older video/software version. The process was load a sketch to the arduino (teensy) on the arduino program then open your software and connect it. So if thats not the correct process now. Thats probably the issue.

**Comment by TylerTimoJ** [comment]
That's still the right process. I'm thinking the Arduino code might not be compatible with the current version of the software, if you could copy and paste your arduino code here and let me take a look at it.

**Comment by MAXimumEFFORT843333** [comment]
The code im using is provided in your files? But when i get home ill post it here

**Comment by MAXimumEFFORT843333** [comment]
here is the code i had tried using 

#include "FastLED.h"
#define WIDTH 25
#define HEIGHT 25

const int NUM_LEDS = WIDTH * HEIGHT;

CRGB leds[NUM_LEDS];

void setup() {
  FastLED.addLeds<WS2812B, 6, GRB>(leds, NUM_LEDS);
  Serial.begin(4608000);
}

void loop() {}

void serialEvent() {
  switch (Serial.read()) //read header
  {
    case 0x05: //request for matrix definition
      Serial.println(WIDTH);
      Serial.println(HEIGHT);
      break;

    case 0x11: //frame data
      Serial.readBytes((char*)leds, NUM_LEDS * 3);
      FastLED.show();
      Serial.write(0x06); //acknowledge
      break;
  }
}

**Comment by MAXimumEFFORT843333** [comment]
.........im an idiot please disregard




didn't change which pint he data cable was on..... go figure..... sorry

**Comment by TylerTimoJ** [comment]
No worries!
I just recreated your setup as best I could, a 16x16 ws2812b matrix on a teensy 3.5, and it worked so I figured the data pin would be the next thing to check. I did discover a new bug though. My matrix has is vertical, scan, and the first pixel is the bottom left, but it looks like setting the origin to top right gets does what I'd expect setting it to bottom left would.

Guess I'm going to have to look into the pixel order code again...
